miniwallet/deploy/002_deploy_miniID.ts

```diff
@@ -0,0 +1,55 @@
+import config from '../config'
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+import { DeployFunction } from 'hardhat-deploy/types'
+// import { ethers } from 'hardhat'
+
+const deployFunction: DeployFunction = async function (
+  hre: HardhatRuntimeEnvironment
+) {
+  const { deployments, getNamedAccounts, getChainId } = hre
+  const { deploy } = deployments
+  const { deployer, operatorA } = await getNamedAccounts()
+  const chainId = await getChainId()
+
+  console.log(`chainId: ${chainId}`)
+
+  const deployedMiniID = await deploy('MiniID', {
+    contract: 'MiniID',
+    from: deployer,
+    proxy: {
+      proxyContract: 'ERC1967Proxy',
+      proxyArgs: ['{implementation}', '{data}'],
+      execute: {
+        init: {
+          methodName: 'initialize',
+          args: []
+        }
+      }
+    },
+    log: true
+  })
+
+  const miniID = await hre.ethers.getContractAt('MiniID', deployedMiniID.address)
+
+  console.log('MiniID deployed to  :', miniID.address)
+  console.log('MiniID Name         : ', await miniID.name())
+  console.log('MiniID Symbol       : ', await miniID.symbol())
+  // Mint test tokens
+  if (chainId !== '1666600000,') {
```

Comment:
extra `,`

Reply:
Refactored approach and commit are [here](https://github.com/polymorpher/sms-wallet/issues/13#issuecomment-1246191757)